com.unity.formats.fbx/Documentation~/integration.md

```diff
@@ -4,8 +4,9 @@ Use the Unity Integration tool in **Autodesk® Maya®**, **Autodesk® Maya LT™
 
 | Section | Description |
 | :--- | :--- |
-| [Set up Unity Integration for your 3D modeling application](integration-setup.md) | Set up Unity Editor integration with **Autodesk® Maya®**, **Autodesk® Maya LT™**, or **Autodesk® 3ds Max®**. |
+| [Set up Unity Integration for your 3D modeling application](integration-setup.md) | Set up Unity Editor integration with Autodesk® Maya®, Autodesk® Maya LT™, or Autodesk® 3ds Max®. |
 | [Use the Unity Integration tool in Maya](integration-maya-roundtrip.md) | Work in Maya to import FBX files from a Unity project and re-export them after editing. |
 | [Manage Export Sets in Maya](integration-maya-manage-export-sets.md) | Create Export Sets to export Maya objects that are not part of an existing Export Set. Edit export options of existing Export Sets. |
 | [Use the Unity Integration tool in 3ds Max](integration-3ds-max-roundtrip.md) | Work in 3ds Max to import FBX files from a Unity project and re-export them after editing. |
 | [Manage objects to export from 3ds Max](integration-3ds-max-manage-objects-to-export.md) | Manage the list of 3ds Max objects to export within Selection Sets. |
+| [Convert Options window](ref-convert-options.md) | Get the description of all settings available in the Convert Options window that displays when you perform a conversion to an FBX Prefab Variant. |
```

Comment:
This "Convert Options window" reference page is currently NOT a child of this page. Normal or mistake?

Reply:
Ha! 😅 Clearly a mistake, I don't know what happened, this table line should even be in the Prefab Variants landing page, where it is currently missing. You have better eyes than me – thanks!